core/chains/evm/txmgr/orm.go

```diff
@@ -263,3 +278,23 @@ func loadEthTxesAttemptsReceipts(q pg.Queryer, etxs []*EthTx) (err error) {
 	}
 	return nil
 }
+
+func loadConfirmedAttemptsReceipts(q pg.Queryer, attempts []EthTxAttempt) ([]EthTxAttempt, error) {
+	attemptHashM := make(map[common.Hash]*EthTxAttempt, len(attempts))
+	attemptHashes := make([][]byte, len(attempts))
```

Comment:
I think it would help readability to leave out `attempt` from the var names:
```suggestion
	byHash := make(map[common.Hash]*EthTxAttempt, len(attempts))
	hashes := make([][]byte, len(attempts))
```

Reply:
Or some kind of abbreviation for `attempt`... :thinking:

Reply:
this makes sense, byHash is better i think